app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/group/OrganisationDetails.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.models.registration.group
+
+import play.api.libs.json.{Json, OFormat}
+
+case class OrganisationDetails(organisationType: Option[String] = None, organisationName: String)
```

Comment:
The api can have optional org type but do we in the front end?

Reply:
We probably could have it as a String.   This FE model is a bit different as it's not getting populated by filling in a form, so I guess we can define it more tightly.

For MVP I guess it will always be "organisationType" : "UkCompany"